### ISSUE TITLE
Is there any reason not to train with the map flag?

### ISSUE THREAD

**Issue by willbattel** [AlexeyAB/darknet#2863]
Is there any reason not to train with the `-map` flag? I know it adds the mAP to the chart but I don’t see why it would ever make sense not to use it- and therefore why it wouldn’t just be enabled by default.

**Comment by AlexeyAB** [comment]
If you do not have a Validation dataset to test the mAP, and if you don’t want to test the mAP in the Training dataset (it can be very long).

**Comment by willbattel** [comment]
Okay, that's what I figured but since I have a validation set and mAP calculation isn't taking very long I have it enabled.
